OpenRA.Platforms.Default/OpenAlSoundEngine.cs

```diff
@@ -102,6 +102,8 @@ public OpenAlSoundEngine()
 					throw new InvalidOperationException("Can't create OpenAL device");
 			}
 
+			Game.OnQuit += () => { ALC10.alcCloseDevice(device); };
```

Comment:
This is already done when disposing the sound engine.  If the sound engine isn't being disposed then that needs to be fixed – don't just force close it again!